app/src/main/java/com/gdg/android/MainActivity.kt

```diff
@@ -123,10 +138,171 @@ fun ProfileScreen(name: String, major: String, modifier: Modifier = Modifier) {
     }
 }
 
+@Composable
+fun LoginPage(navController: NavController? = null) {
+    val context = LocalContext.current
+
+    val majorTextValue = remember { mutableStateOf("") }
+    val nameTextValue = remember { mutableStateOf("") }
```

Comment:
by에는 getter, setter가 내장되어 있어 =보다 훨씬 편리합니다. =을 사용하면 .value로 값에 접근해야되지만 by를 사용하면 변수명으로 바로 값에 접근할 수 있습니다.
```suggestion
    val majorTextValue by remember { mutableStateOf("") }
    val nameTextValue by remember { mutableStateOf("") }
```